README.md

```diff
@@ -281,6 +288,13 @@ Supported domains: BLAS, LAPACK, RNG
             <td align="center">Dynamic, Static</td>
             <td align="center">DPC++</td>
         </tr>
+        <tr>
+            <td rowspan=1 align="center">DFT</td>
+            <td align="center">Intel GPU</td>
+            <td rowspan=1 align="center">Intel(R) oneAPI Math Kernel Library</td>
+            <td align="center">Dynamic, Static</td>
+            <td align="center">DPC++</td>
+        </tr>
```

Comment:
Not sure if we should mention that Windows is supported here if we are not testing it internally?

Reply:
Sure, I have removed that.